packages/amplify_core/lib/src/types/notifications/push/push_notification_message.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/amplify_core.dart';
+
+class PushNotificationMessage with AWSDebuggable {
+  String? messageId;
+  String? senderId;
+  DateTime? sentTime;
+
+  PushNotificationMessageContent? content;
+  Map<Object?, Object?>? data;
+
+  PushNotificationMessage({
```

Comment:
Add doc. (to public properties as well) The linter is not enabled for amplify_core atm, but we should follow the best practices of the codebase :) 

Class properties goes after the constructor.